### PR TITLE
Add guidelines

### PR DESCRIPTION
**Pull request by thomasarsouze**

### CODE REVIEW

File: doc/development/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+#. The theme we are using is the `Read the Docs Sphinx Theme <https://sphinx-rtd-theme.readthedocs.io/en/stable/>`_.
+   The theme can be installed like this
+   ::
+
+       pip install sphinx_rtd_theme
+
+   Once you've installed the theme, write in your ``conf.py`` file
+   ::
+
+       html_theme = "sphinx_rtd_theme"
```
[the remark this answers is not in the document]
Comment (thomasarsouze) [reply doc/development/contributing.rst:139]: We have a gallery here: https://sphinx-themes.org/

I didn't modify the choice of the theme, but I agree that I like `pydata-sphinx-theme` better

File: doc/development/contributing.rst [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+Documentation
+-------------
+
+You can also contribute to the documentation. If you find some parts that are not explained enough or uncleared, you can complete or
+improve the documentation.
+
+Once you have forked the package on your device, you have to install Sphinx_ to generate the HTML output
+::
+
+    pip install sphinx
```
[the remark this answers is not in the document]
Comment (thomasarsouze) [reply doc/development/contributing.rst:111]: Didn't modify this section. I would stick to `pip` as most sphinx themes only have a `pip` install available

File: doc/development/guidelines.md [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Topic :: Scientific/Engineering",
+]
+dynamic = ["version"]
+dependencies = [
```
[the remark this answers is not in the document]
Comment (thomasarsouze) [reply doc/development/guidelines.md:117]: I don't know... I think this section is usefull for develop mode, but I don't know how to link that to the `meta.yaml`.

[the remark this answers is not in the document]
Comment (thomasarsouze) [reply doc/development/guidelines.md:117]: Which makes me think that we have no guidelines for the `meta.yaml`

[the remark this answers is not in the document]
Comment (pradal) [reply doc/development/guidelines.md:117]: There are implcit guidelines that came from conda-forge.
We can latter speaks about it.

**Review by thomasarsouze — COMMENTED** [review]
> Add perhaps some example packages. Then people can copy/past 

Agree. We should update `openalea.mtg` for exemple and mention this as an openalea "template"

**Pull request merged**